TMS.ConsoleApp/Entities/Lecture.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TMS.ConsoleApp.Entities
+{
+    internal class Lecture : EntityBase
+    {
+        public string Topic { get; set; } = string.Empty;
```

Comment:
PracticalLesson field is not necessary, Lecture and TrainingLecture are different separate not linked each to each classes